### PR TITLE
[new] Integrate mst  and grafana 

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

We have instructions for connecting Timescale Cloud with Grafana.
Added this procedure to add MST database as a data source in MST Grafana.

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: mst/grafana-mst.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: Integrate Managed Service for TimescaleDB as a data source in Grafana
+excerpt: Integrate Managed Service for TimescaleDB as a data source in Grafana to visualize your data
+keywords: [Grafana, visualizations, analytics]
+---
+
+# Integrate Managed Service for TimescaleDB and Grafana
+
+You can integrate the services within the Managed Service for TimescaleDB platform
+to tranform into a true data cloud. Grafana ships with built-in Prometheus,
```
Comment (Loquacity) [inline mst/grafana-mst.md:10]: This sentence doesn't feels like it doesn't really say anything. Can we say something more meaningful, like "You can use Managed Service for TimescaleDB with Grafana to visualize your data" or similar? Let's get to what's actually happening in this section.

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: mst/grafana-mst.md [hunk 2]
```diff
@@ -6,10 +6,9 @@ keywords: [Grafana, visualizations, analytics]
 
 # Integrate Managed Service for TimescaleDB and Grafana
 
-You can integrate the services within the Managed Service for TimescaleDB platform
-to tranform into a true data cloud. Grafana ships with built-in Prometheus,
-PostgreSQL, Jaeger, and other data source plugins that allow you to query and
-visualize data from a compatible database.
+You can integrate Managed Service for TimescaleDB with Grafana service to visualize your data.
```
Comment (Loquacity) [inline mst/grafana-mst.md:9]:
```suggestion
You can integrate Managed Service for TimescaleDB with Grafana to visualize your data.
```

Much better! 💖

**Review by Loquacity — APPROVED** [review]

**Review by alexthinath — APPROVED** [review]

File: mst/grafana-mst.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Before you begin
+
+*   Create a Managed Service for TimescaleDB [service][mst-install]
+*   Create a service for [Grafana in MST][grafana-install]
+
+### Configuring Managed Service for TimescaleDB as a data source
+
+<procedure>
+
```
Comment (charislam) [inline mst/grafana-mst.md:21]:
```suggestion
## Configure Managed Service for TimescaleDB as a data source

<procedure>

### Configuring Managed Service for TimescaleDB as a data source

```

Would add some introductory text between the heading and procedure as well

File: mst/grafana-mst.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+1.  In the [MST account][mst-login] `Services` view, click the Managed Service
+    for TimescaleDB service that you want to add as a data source for the Grafana
+    service.
+1.  In the `Overview` tab for the service go to `Service Integrations` section.
```
Comment (charislam) [inline mst/grafana-mst.md:25]:
```suggestion
1.  In the `Overview` tab for the service go to the `Service Integrations` section.
```

File: mst/grafana-mst.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+1.  Click the `Set up integration` button.
+1.  In the `Available service integrations for TimescaleDB` dialog, click
+    the `Use Integration` button for `Datasource`.
+1.  In the dialog that appears choose the Grafana service in the drop-down menu,
```
Comment (charislam) [inline mst/grafana-mst.md:29]:
```suggestion
1.  In the dialog that appears, choose the Grafana service in the drop-down menu,
```

File: mst/grafana-mst.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    and click the `Enable` button.
+1.  In the `Services` view, click the Grafana service to which you added the MST
+    service as a data source.
+1.  In the `Overview` tab for the Grafana service, make a note of the `User` and
+    `Password` fields.
+1.  In the `Overview` tab for the Grafana service, click the link in the
+   `Service URI` field to open Grafana.
+1.  Log in to Grafana with your service credentials.
+1.  Navigate to `Configuration` → `Data sources`. The data sources page lists
+    Managed Service for TimescaleDB as configured data sources for the Grafana instance.
```
Comment (charislam) [inline mst/grafana-mst.md:39]:
```suggestion
    Managed Service for TimescaleDB as a configured data source for the Grafana instance.
```

File: mst/grafana-mst.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+</procedure>
+
+When you have configured Managed Service for TimescaleDB as a data source in
+Grafana, you can create panels that are populated with data using SQL.
```
Comment (charislam) [inline mst/grafana-mst.md:44]: Might be nice to link to the Grafana tutorials

**Review by charislam — APPROVED** [review]

**Pull request merged**